RELEASE_NOTES.rst

```diff
@@ -174,9 +174,10 @@ In this release, coala-bears has been revamped with new bears and more configs
 are added for existing bears.
 
```

Comment:
Please use present tense (Adds) in commit body and end sentences with full stops